ERCS/erc-_____.md

```diff
@@ -0,0 +1,342 @@
+---
+eip: ____
```

Comment:
```suggestion
eip: 7798
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.